opendcs-rest-api/src/main/webapp/swaggerui/swagger-initializer.js

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2025 OpenDCS Consortium and its Contributors
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+window.onload = function() {
+  //<editor-fold desc="Changeable Configuration Block">
+
+  // the following lines will be replaced by docker/configurator, when it runs in a docker-container
+  window.ui = SwaggerUIBundle({
+    url: "../openapi.json",
```

Comment:
Is the jaxrs swagger generating this on build or providing an endpoint to query?

Reply:
More like the answer is both. Zack added a gradle task to generate locally so that you don't have to spin up the web service to create the openapi during development. At runtime though there is an endpoint to query which is provided by the `OpenApiResource` class.